src/schema/nmdc.yaml

```diff
@@ -157,15 +157,15 @@ classes:
       - CHMO:0000470
     slots:
       - eluent_introduction_category
-      - has_calibration
+      - generates_calibration
       - has_chromatography_configuration
       - has_mass_spectrometry_configuration
     slot_usage:
       id:
         structured_pattern:
           syntax: "{id_nmdc_prefix}:(dgms|omprc)-{id_shoulder}-{id_blade}$"
           interpolated: true
-      has_calibration:
+      generates_calibration:
```

Comment:
Can we move the structured_pattern to the global definition of the slot?  That way the range and structured_pattern are defined in the same spot.